src/linkedin.js

```diff
@@ -40,7 +40,16 @@ const check = response => {
   );
 };
 
-const linkedinGet = (url, accessToken) =>
+const linkedinGetUserDetails = (url, accessToken) =>
+  axios({
+    method: 'get',
+    url: url + '?projection=(id,localizedLastName,localizedFirstName,profilePicture(displayImage~:playableStreams))',
```

Comment:
Would this live better in `getApiEndpoints`?

Reply:
done :)